.eslintrc.js

```diff
@@ -39,6 +39,13 @@ module.exports = {
         'no-console': 'error',
         // Allow unary ++ and -- operators
         'no-plusplus': 'off',
+        // Allow up to 8 props, but mostly rely on max line length to limit this
+        'object-curly-newline': ['error', {
+            ObjectExpression: { minProperties: 8, multiline: true, consistent: true },
+            ObjectPattern: { minProperties: 8, multiline: true, consistent: true },
+            ImportDeclaration: { minProperties: 8, multiline: true, consistent: true },
+            ExportDeclaration: { minProperties: 8, multiline: true, consistent: true },
+        }],
```

Comment:
This is a base linting change we've been discussing for a while in SlipStream.  A few common examples where we are just using simple destructuring but forced to make the code arguably less readble would be:

```
// Actions in a Vuex store - bad
[ACTION_NAME]({ commit }, {
    arg1,
    arg2,
    arg3,
    arg4,
}) {

// Actions in a Vuex store - good
[ACTION_NAME]({ commit }, { arg1, arg2, arg3, arg4 }) {
```

```
// Short imports from lodash - bad
import {
    get,
    find,
    flatMap,
    omitBy,
} from 'lodash-es';

// Short imports from lodash - good
import { get, find, flatMap, omitBy } from 'lodash-es';
```